### ISSUE TITLE
create bridges directly to avoid firewalld

### ISSUE THREAD

**Issue by cagney** [libreswan/libreswan#1745]
see https://gitlab.com/libvirt/libvirt/-/issues/588#note_1902618527

**Comment by cagney** [comment]
this is quick:
```
+       /usr/bin/time sudo nmcli connection add type bridge ifname $* con-name $*
```
but this takes ~13 seconds:
```
+       /usr/bin/time sudo nmcli connection up $*
+       touch $@
```

**Comment by cagney** [comment]
upstream seem to have made "open" D.T.R.T., so perhaps not necessary once commit arrives

**Comment by cagney** [comment]
should be fixed in libvirt 10.8.0

**Comment by cagney** [comment]
f41 shipped with 10.6 (rawhide has 10.9) so still waiting.
But perhaps, per @antonyantony 's comment https://github.com/libreswan/libreswan/issues/1291#issuecomment-2433783995 perhaps it should be done regardless.  One less dependency.
